src/api/scripts/test_sql_models.py

```diff
@@ -0,0 +1,300 @@
+from api.models import (
```

Comment:
These are great... are you thinking we'll remove the MV versions down the road? If so we should create a ticket for that, so we don't forget. Also we already merged in the SQS code which we're not (yet) using... all fine, just don't want a lot of code hanging around that we're not using.

Reply:
we can chat about this further